packages/composites/bob/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@chainlink/bob",
```

```suggestion
  "name": "@chainlink/bob-adapter",
```
Our current convention to the suffix the package with `-adapter`.